### PR TITLE
Include extensions-sdk docs from upstream repo

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@gmail.com>

### Proposed changes

Add first version of extensions SDK docs from upstream repo
* Add TOC sections under /manual/desktop, next to "extensions"
* Add TOC sections under /reference (note there are links between pages from the "manual" docs to the "reference" docs 

This can be tested locally with `docker compose up -d --build`

Screenshot 2022-05-04 at 10 11 13

### CODE REVIEW

File: _data/toc.yaml [hunk 1]
```diff
@@ -1190,6 +1240,58 @@ manuals:
       title: Dev Environments (Preview)
     - path: /desktop/extensions/
       title: Extensions (Preview)
+    - sectiontitle: Extensions SDK (Preview)
```
Comment (aevesdocker) [inline _data/toc.yaml:1243]: Just checking if this should be changed from 'Preview' to 'Beta'. I have another PR to be merged tomorrow that changes the user manual docs to 'Beta'.

Comment (gtardif) [reply _data/toc.yaml:1243]: I can change both in this PR, to avoid conflicts...

Comment (aevesdocker) [reply _data/toc.yaml:1243]: That'd be great, thank you.

Comment (gtardif) [reply _data/toc.yaml]: Also mentioned Beta in the reference section

**Review by aevesdocker — APPROVED** [review]
LGTM

**Comment by gtardif** [comment]
Looking at broken anchors...

**Pull request merged**